dockerfile

```diff
@@ -0,0 +1,47 @@
+# Use python:buster image as base
+FROM python:buster as base
+
+# Update pip
+RUN apt-get update && pip install --upgrade pip
+RUN apt-get install -y curl
+
+# Install Poetry and add to path
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+ENV PATH = "${PATH}:/root/.poetry/bin"
+
+FROM base as production
+
+# Expose Port 8000
+EXPOSE 8000
+
+# Copy code accross
+COPY . /usr/DevOps-Course-Starter
+
+WORKDIR /usr/DevOps-Course-Starter
+
+# Install poetry dependencies 
+RUN poetry install
+
+# To-do app entrypoint
+
+ENTRYPOINT ["poetry","run","gunicorn", "-w", "4","--bind","0.0.0.0", "todo_app.app:create_app()"]
+
+FROM base as development
+
+# Expose port 5000
+EXPOSE 5000
+
+# Copy requirements
+WORKDIR /usr/DevOps-Course-Starter
+COPY pyproject.toml /usr/DevOps-Course-Starter
```

Comment:
You should copy the poetry.lock file as well to control versions more specifically.